### DISCUSSION TITLE
Suddenly getting "a CNAME record with the same name already exists in this zone" via Acmebot:Webhook

### DISCUSSION THREAD

**Discussion by davetapley** [polymind-inc/acmebot#704] [Q&A] [answered]
This has been working create for a few years, but since April 18th I get this message just after UTC midnight via the `Acmebot:Webhook`:

```
Acmebot
IssueCertificate
Reason
Microsoft.Azure.WebJobs.Extensions.DurableTask.FunctionFailedException:
The activity function 'Dns01Authorization' failed:
"The record could not be created because a CNAME record with the same name already exists in this zone.
Status: 409 (Conflict)
```
[full message here](https://gist.github.com/davetapley/627965624b7639ed23a48f50d1c5ec16).

We didn't make any DNS changes, so I'm not sure what's going on.

I see the same error here, but no follow up (@uniqueumang do you remember what happened? 🙏🏻 )
- https://github.com/shibayan/keyvault-acmebot/issues/173

I would appreciate any tips on how to diagnose. Looking in Azure portal I don't know where to start, and everything looks normal on the Acmebot dashboard (although no certs have renewed since April 18th, because none are in the renew window yet.)

**Comment by shibayan** [comment, marked as answer]
The error message tells you that a CNAME record with the same name as the TXT record you are trying to create already exists, so check to see if a CNAME exists with the record name included in the error message.

**Reply by davetapley** [reply]
Luckily I was able to track it down by trial and error, but I don't see either the domain name, or a record name in the error which came via `Acmebot:Webhook`, is there somewhere else to look?
